CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.0.0-rc.2 (2019/10/11)
+
+- Fixed border, font, and hover styling issues. [#11](https://github.com/blackbaud/skyux-ag-grid/pull/11)
+- Added an autocomplete valueFormatter. [#11](https://github.com/blackbaud/skyux-ag-grid/pull/11)
```

Comment:
I'd format `valueFormatter` as a code snippet and add "function" after it. (I think based on the code that this is a function, but let me know if I got this wrong.)

Reply:
oh yeah it is, I'll do that real quick!